src/components/results/sensitivityAnalysis/paged-sensitivity-analysis-result.js

```diff
@@ -0,0 +1,172 @@
+            });
+        }
+
+        return baseFilters;
+    }, [intl, sensiKindIndex, nOrNkIndex, result]);
+
+    const { updateFilter, getFilterSelector } = useRowFilter(
+        DATA_KEY_TO_FILTER_KEY
+    );
+
+    // Add default sort on sensitivity col
+    const colKey = 'SENSITIVITY';
+    const sortWay = SORT_WAYS.desc;
+    const { onSortChanged, sortSelector } = useAgGridSort(
+        DATA_KEY_TO_SORT_KEY,
+        { colKey: colKey, sortWay }
+    );
+
+    const { snackError } = useSnackMessage();
+
+    const handleChangePage = useCallback((_, newPage) => {
+        setPage(newPage);
+    }, []);
+
+    const handleChangeRowsPerPage = useCallback((event) => {
+        setRowsPerPage(parseInt(event.target.value, 10));
+        setPage(0);
+    }, []);
+
+    const fetchResult = useCallback(() => {
+        setIsLoading(true);
+
+        const selector = {
+            isJustBefore: !nOrNkIndex,
+            functionType: FUNCTION_TYPES[sensiKindIndex],
+            offset: page * rowsPerPage,
+            chunkSize: rowsPerPage,
+            ...getFilterSelector(),
+            ...sortSelector,
+        };
+
+        fetchSensitivityAnalysisResult(studyUuid, nodeUuid, selector)
+            .then((res) => {
+                setResult(res);
+
+                const { filteredSensitivitiesCount = 0 } = res;
+                setCount(filteredSensitivitiesCount);
+            })
+            .catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'PagedSensitivityResult',
+                });
+            })
+            .finally(() => {
+                setIsLoading(false);
+            });
+    }, [
+        snackError,
+        nOrNkIndex,
+        sensiKindIndex,
+        studyUuid,
+        nodeUuid,
+        page,
+        rowsPerPage,
+        getFilterSelector,
+        sortSelector,
+    ]);
+
+    useEffect(() => {
+        fetchResult();
+    }, [fetchResult]);
+
+    return (
+        <>
+            {isLoading && <LinearProgress />}
+            <FilterPanel filtersDef={filtersDef} updateFilter={updateFilter} />
+            <SensitivityAnalysisResult
+                result={result?.sensitivities || []}
+                nOrNkIndex={nOrNkIndex}
+                sensiToIndex={sensiKindIndex}
+                onSortChanged={onSortChanged}
+            />
+            <TablePagination
+                component="div"
+                rowsPerPageOptions={PAGE_OPTIONS}
+                colSpan={3}
+                count={count}
+                rowsPerPage={rowsPerPage}
+                page={page}
+                showFirstButton={true}
+                showLastButton={true}
+                onPageChange={handleChangePage}
+                onRowsPerPageChange={handleChangeRowsPerPage}
+                ActionsComponent={TablePaginationActions}
+            >
+                <span>{isLoading ? 'waiting' : 'finished'}</span>
```

Comment:
is it needed ? sounds like a weird behavior

Reply:
I added the circular loader, so there is no need for this now. thank you